runtime/jcl/common/jclcinit.c

```diff
@@ -146,14 +149,20 @@ jint computeFullVersionString(J9JavaVM* vm)
 	#define OMR_INFO ""
 #endif /* J9VM_GC_MODRON_GC */
 
+#if defined(OPENJDK_TAG) && defined(OPENJDK_SHA)
+	#define OPENJDK_INFO "\nOpenJDK  - " OPENJDK_SHA " based on " OPENJDK_TAG
```

Comment:
From an IBM perspective, I don't think the "OpenJDK" should be hard coded, but passed in as another define. For IBM internal builds it should be configurable since I expect they would use "Oracle" instead of "OpenJDK".

Also, based on the -version discussion on Slack today I'm not sure we need or want to include the OPENJDK_TAG along with the SHA, as this piece of information should be elsewhere in the -version output.

Reply:
I'd be ok with hard coding "JCL".